generators/app/templates/scripts/_rollup.config.js

```diff
@@ -9,9 +9,15 @@ const resolve = require('rollup-plugin-node-resolve');
 const {uglify} = require('rollup-plugin-uglify');
 const {minify} = require('uglify-es');
 const pkg = require('../package.json');
+
+/* General Globals */
+const moduleName = '<%= moduleName %>';
+const pluginName = '<%= pluginName %>';
```

Comment:
Mostly I added all the main variables at the top of the file, so that users won't have to dig into the actual builds unless they really need to.